akka-javasdk/src/main/scala/akka/javasdk/impl/consumer/ReflectiveConsumerRouter.scala

```diff
@@ -24,30 +25,28 @@ private[impl] class ReflectiveConsumerRouter[A <: Consumer](
     ignoreUnknown: Boolean)
     extends ConsumerRouter[A](consumer) {
```

Comment:
In a separate PR you could try to remove ConsumerRouter. I don't think we need both ConsumerRouter  and ReflectiveConsumerRouter